packages/changed-elements-react/src/clients/iTwinApi.ts

```diff
@@ -31,7 +31,11 @@ export async function callITwinApi(args: CallITwinApiParams): Promise<Record<str
     await throwBadResponseCodeError(response, "iTwin API request failed.");
   }
 
-  return response.json();
+  try {
+    return response.json();
+  } catch (error) {
+    return undefined;
+  }
```

Comment:
Is there a way around doing this as a try/catch? What case causes `.json()` to fail?